packages/plugins/users-permissions/admin/src/components/FormModal/index.js

```diff
@@ -66,7 +66,9 @@ const FormModal = ({
                               {...input}
                               error={errors[input.name]}
                               onChange={handleChange}
-                              value={values[input.name]}
+                              value={
+                                input.name === 'noName' ? values.redirectUri : values[input.name]
```

Comment:
I've also fixed the textfield with "The redirect URL to add in your PROVIDER application configurations". So it shows the correct value from backend.